ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -499,6 +514,18 @@ - (void)webView:(__unused WKWebView *)webView decidePolicyForNavigationAction:(W
   else {
     decisionHandler(WKNavigationActionPolicyAllow);
   }
+  
+  if (navigationAction.navigationType == WKNavigationTypeLinkActivated) {
+    // Skip scroll handler if user navigate to an anchor link
+    webView.scrollView.delegate = nil;
```

Comment:
@kazkondo Please check this part. I have to disable/enable scroll event handler manually because there is no callback called after this timing

Reply:
What should I check?